app/models/signon_identity.rb

```diff
@@ -33,7 +33,7 @@ def from_session_data(data)
 
     def find_or_create_authorized_user(info)
       email = info.fetch('email')
-      sso_orgs = valid_pvb_orgs(info)
+      sso_orgs = info.fetch('permissions').map { |p| p.fetch('organisation') }
 
       unless sso_orgs.any?
```

Comment:
The overall validation for the estate is that the signon class checks that there is at least an organisation and a new controller before action checks that the current estate is valid: https://github.com/ministryofjustice/prison-visits-2/pull/478/commits/732f575e3307beb210cadebdc830bd8a6a22c9c4#diff-55c5b7aecfb519d0e4880eaf2788eb6eR67